src/lib/seam/components/DeviceDetails/DeviceDetails.tsx

```diff
@@ -36,12 +37,13 @@ export function DeviceDetails({
     return null
   }
 
-  const props = {
+  const props: Omit<DeviceDetailsProps, 'deviceId'> = {
     disableLockUnlock,
     disableDeleteAccessCode,
     disableResourceIds,
     disableCreateAccessCode,
     disableEditAccessCode,
+    disableScheduledClimates,
```

Comment:
```suggestion
    disableClimateSettingSchedules,
```

I suspect we will deprecate this  flip the default value to true in the next major version after we introduce `disableScheduledClimateSettings`.

Reply:
There is no Seam API concept called "scheduled climates". Seam only provides "climate setting schedules" which will eventually be deprecated in favor of "scheduled climate settings". Thus, we should name this prop `disableClimateSettingSchedules` to match the Seam API resource. 

When we reach the deprecation stage, we will change the default value of `disableClimateSettingSchedules` to false an introduce `disableScheduledClimateSettings` instead.

Reply:
Gotcha—thank you for the explanation!!